Cargo.toml

```diff
@@ -7,3 +37,9 @@ members = [
     "crates/validation",
     "crates/validation/macros"
 ]
+
+[workspace.dependencies]
+anyhow = "1.0.66"
+aurae-proto = {path = "./aurae-proto", version = "0.1.0"}
```

Comment:
Not sure there is any effect from `version` when using `path`

Reply:
probably not.. i'm just copying what was there before.